Sources/armory/logicnode/GetVelocityNode.hx

```diff
@@ -16,11 +16,12 @@ class GetVelocityNode extends LogicNode {
 #if arm_physics
 		var rb: RigidBody = object.getTrait(RigidBody);
 
-		if (rb != null) {
-			if (from == 0) return rb.getLinearVelocity();
-			else if (from == 1) return rb.getAngularVelocity();
+		return switch (from) {
+			case 0: rb.getLinearVelocity();
+			case 1: rb.getAngularVelocity();
+			default: null;
 		}
 #end
-		return null;
```

Comment:
If the `arm_physics` flag is not set this function won't return anything anymore.

Reply:
Just a doubt here: the `default: null;` line of the switch is the same as if i put `return null;` without switch? This is why i deleted those lines.

In the case of `arm_physics`, i would not remove the `return`, but and for other cases? If i found some thing like that i send here to clarify this

Reply:
In would be the same if there wasn't the `#end` in between. The removed `return null` was executed independently of the `arm_physics` flag (https://haxe.org/manual/lf-condition-compilation.html), now you put it inside the block that's only included in the build if that flag is set. What you can do is to replace `#end` by

```haxe
#else
	return null;
#end
```